extensions/notebook/src/book/bookTocManager.ts

```diff
@@ -454,7 +450,7 @@ export class BookTocManager implements IBookTocManager {
 		this._movedFiles = files;
 	}
 
-	public set originalToc(files: Map<string, JupyterBookSection[]>) {
+	public set originalToc(files: Map<string, string>) {
```

Comment:
Seems like you could just get rid of this and make tocFiles public (why is it called originalToc anyways?)